src/test/python_tests/test_path_specialization.py

```diff
@@ -91,8 +103,66 @@ def test_path():
     assert_that(actual, is_(False))
 
 
+def test_args_recursive():
+    """Test formatting using --recursive in args."""
+
+    init_params = copy.deepcopy(defaults.VSCODE_DEFAULT_INITIALIZE)
```

Comment:
what is "VSCODE_DEFAULT_INITIALIZE" ?

Reply:
it is the default values that VSCode uses to initialize a language server.